dbt/adapters/hive/impl.py

```diff
@@ -29,12 +29,14 @@
 from dbt.adapters.hive import HiveColumn
 from dbt.adapters.base import BaseRelation
 from dbt.clients.agate_helper import DEFAULT_TYPE_TESTER
-from dbt.logger import GLOBAL_LOGGER as logger
 from dbt.utils import executor
 
 import time
 from dbt.clients import agate_helper
 
+from dbt.events import AdapterLogger
+logger = AdapterLogger("Impala")
```

Comment:
typo, please change to "Hive"

Reply:
Looks good to me if this is fixed. Hoping PR#20 and PR#12 doesn't conflict as there are some formatting fixes in here, probably touching the same lines that were edited in PR#20.